rules/linux/execution_shell_evasion_linux_binary.toml

```diff
@@ -0,0 +1,104 @@
+administrator. It indicates a potentially malicious actor attempting to improve the capabilities or stability of their
+access.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Restricted Shell Breakout via  Linux Binary(s)"
+references = [
+    "https://gtfobins.github.io/gtfobins/apt/",
+    "https://gtfobins.github.io/gtfobins/apt-get/",
+    "https://gtfobins.github.io/gtfobins/nawk/",
+    "https://gtfobins.github.io/gtfobins/mawk/",
+    "https://gtfobins.github.io/gtfobins/awk/",
+    "https://gtfobins.github.io/gtfobins/gawk/",
+    "https://gtfobins.github.io/gtfobins/busybox/",
+    "https://gtfobins.github.io/gtfobins/c89/",
+    "https://gtfobins.github.io/gtfobins/c99/",
+    "https://gtfobins.github.io/gtfobins/cpulimit/",
+    "https://gtfobins.github.io/gtfobins/crash/",
+    "https://gtfobins.github.io/gtfobins/env/",
+    "https://gtfobins.github.io/gtfobins/expect/",
+    "https://gtfobins.github.io/gtfobins/find/",
+    "https://gtfobins.github.io/gtfobins/flock/",
+    "https://gtfobins.github.io/gtfobins/gcc/",
+    "https://gtfobins.github.io/gtfobins/mysql/",
+    "https://gtfobins.github.io/gtfobins/nice/",
+    "https://gtfobins.github.io/gtfobins/ssh/",
+    "https://gtfobins.github.io/gtfobins/vi/",
+    "https://gtfobins.github.io/gtfobins/vim/",
+    "https://gtfobins.github.io/gtfobins/capsh/",
+    "https://gtfobins.github.io/gtfobins/byebug/",
+]
+risk_score = 47
+rule_id = "52376a86-ee86-4967-97ae-1a05f55816f0"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Execution", "GTFOBins"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and
+
+  /* launch shells from unusual process */
+  (process.name == "capsh" and process.args == "--") or
+
+  /* launching shells from unusual parents or parent+arg combos */
+  (process.name in ("bash", "sh", "dash","ash") and
+    (process.parent.name == "byebug") or
+    (process.parent.name == "crash" and process.parent.args == "-h") or
+
+    /* shells specified in parent args */
+    /* nice rule is broken in 8.2 */
+    (process.parent.args in ("/bin/sh", "/bin/bash", "/bin/dash", "/bin/ash", "sh", "bash", "dash", "ash") and
+        (process.parent.name == "nice") or
+        (process.parent.name == "cpulimit" and process.parent.args == "-f") or
+        (process.parent.name == "find" and process.parent.args == "-exec" and process.parent.args == ";") or
+        (process.parent.name == "flock" and process.parent.args == "-u" and process.parent.args == "/")
+    ) or
+
+    /* shells specified in args */
+    (process.args in ("/bin/sh", "/bin/bash", "/bin/dash", "/bin/ash", "sh", "bash", "dash", "ash") and
+        (process.name == "sensible-pager" and process.parent.name in ("apt", "apt-get") and process.parent.args == "changelog") or
```

Comment:
this applies to _any_ pager and so should be broken out from apt as a parent and without the parent args

Reply:
Breaking this out from apt as a parent creates a lot of noise
![image](https://user-images.githubusercontent.com/91139415/168600154-b0257b9e-789d-417d-9b2c-6395c010f70c.png)
Events that do not invoke shell are also part of the alert when we don't tie it with the process parent.
![image](https://user-images.githubusercontent.com/91139415/168600370-d5070494-7228-43d0-a4e4-5199b4a09d3d.png)

This discourages the user from analyzing further, are we ok inducing so much noise and false positives? 

This optimization method gives scopes to scale with diff parent processes with the child as sensible-pager and that should be ok without deviating from the philosophy of the rule writing.

cc @DefSecSentinel